pkg/shell/shell.css

```diff
@@ -627,6 +627,16 @@ ul.available-interfaces-group {
     vertical-align: top;
 }
 
+/* workaround: make text red for select if contained in has-error, not in patternfly */
+.has-error > .bootstrap-select > button > span {
+    color: #A94442;
+}
+
+
+.containers-run-message {
```

Comment:
I think you should just use the 'hidden' attribute on things you want to hide by default.